Backpack/src/main/java/net/skyscanner/backpack/text/BpkText.kt

```diff
@@ -60,8 +60,9 @@ open class BpkText @JvmOverloads constructor(
     BodyDefault(12),
     Label1(13),
     Label2(14),
-    Footnote(15),
-    Caption(16),
+    Label3(15),
```

Comment:
similarly to last time we changed the enum values here - consumers are using the enum name rather than the value, so this is an internal change and changing the order is not a problem